app/src/bin/crate.in.sh

```diff
@@ -80,26 +80,7 @@ EOF
   else
       JAVA=java
   fi
-  JAVA_VERSION=`$JAVA -version 2>&1 | awk '/version/ {gsub(/"/, "", $3); split($3, parts, ".")} END {print (parts[1] == 1 ? parts[2] : parts[1])}'`
-
-  if [ "x$JAVA_VERSION" = "x" ]; then
-    echo 'Java is missing or $JAVA_HOME is not set correctly. Install a JRE or setup $JAVA_HOME'
-    exit 1
-  fi
```

Comment:
how is this related? shouldn't we still complain when no java is found?

Reply:
You're right that it is a bit unrelated.

Now we get

```
❮ PATH=/usr/bin/ JAVA_HOME=/tmp bin/crate
bin/crate: line 129: exec: java: not found
```

Before it was:

```
❮ PATH=/usr/bin/ JAVA_HOME=/tmp bin/crate       
Java is missing or $JAVA_HOME is not set correctly. Install a JRE or setup $JAVA_HOME
```

I removed it because it was using the `$JAVA_VERSION` to determine if it should be logged and we didn't have the same logic in the `bat` script.

Reply:
Not sure if this is worth re-adding it, just wanted to mention that we somehow change a behaviour slightly. I have no strong opinion here as the `java not found` message may be enough to understand the issue.